packages/@divvi/mobile/src/earn/EarnDepositConfirmationScreen.tsx

```diff
@@ -48,12 +54,22 @@ import { useTokenInfo, useTokenToLocalAmount } from 'src/tokens/hooks'
 import { feeCurrenciesSelector } from 'src/tokens/selectors'
 import type { TokenBalance } from 'src/tokens/slice'
 import { getTokenBalance } from 'src/tokens/utils'
-import { getPreparedTransactionsPossible } from 'src/viem/preparedTransactionSerialization'
+import Logger from 'src/utils/Logger'
+import {
+  getPreparedTransactionsPossible,
+  getSerializablePreparedTransactions,
+} from 'src/viem/preparedTransactionSerialization'
 import {
   getFeeCurrencyAndAmounts,
   type PreparedTransactionsPossible,
 } from 'src/viem/prepareTransactions'
 
+const TAG = 'send/EarnDepositConfirmationScreen'
+const APP_TERMS_AND_CONDITIONS_URL = 'https://valora.xyz/terms'
+const APP_ID_TO_PROVIDER_DOCUMENTS_URL: Record<string, string | undefined> = {
+  beefy: 'https://docs.beefy.finance/',
+}
```

Comment:
This should be returned by the hook (our hooks platform not React hooks 😄). Looks like it was likely done before hooks had this info.
cc @satish-ravi @finnian0826 

Can be addressed in a separate PR too.